misc/helm-charts/testing/environmentd.yaml

```diff
@@ -31,3 +31,19 @@ metadata:
 spec:
   environmentdImageRef: materialize/environmentd:v0.125.0
   backendSecretName: materialize-backend
+  balancerdExternalCertificateSpec:
```

Comment:
does this make cert-manager a pre-req for our sample `Materialize` instance?

Reply:
Ah, I can remove it from the testing yamls.